### ISSUE TITLE
Credentials should not be autofilled by default

### ISSUE THREAD

**Issue by jumde** [brave/brave-ios#507]
### Description: 
Saved credentials are auto-populated.

### Steps to Reproduce 
  1. Navigate to github.com
  2. Login to github.com and save credentials 
  3. Logout and navigate to github.com/login
  4. Credentials will be populated by default.

**Actual result:** 
Credentials are populated by default.

**Expected result:**
Require user interaction to populate credentials

**Reproduces how often:** [Easily reproduced, Intermittent Issue]
Always

**Brave Version:** 
Development (Commit - 60a82d37c7987162971f1d23bf5a8639131a27b5)

**Device details:** 
Simulator (iPhone XS - iOS 12.1)

**Comment by kylehickinson** [comment]
I believe this is how it functions in 1.6

**Comment by jumde** [comment]
This does not block 1.7. I'll tag the issues that are `blocking` from the security standpoint appropriately. For now, I'm filing all `security/privacy` issues  that I run into, let me know if you want me to hold off on filing the `low-pri` issues for now?

**Comment by jhreis** [comment]
@jumde No, please file anything that we would ever want or consider changing. Please just mark them `release/blocking`. (also, if they need to be fixed on this pass, just DM me, so we can get them started ASAP, as we are so close to release).

**Comment by jhreis** [comment]
We need some design elements for this modification, blocking for now.

**Comment by jamesmudgett** [comment]
Instead of Auto-filling credentials lets display the same growl notification seen when opening a link in new tab. 

Message: "Saved credentials found for this site."
Action button should be "Fill".

Reusable UI
![IMG_2841](https://user-images.githubusercontent.com/318119/56238063-2329ba00-6053-11e9-9642-21658217346e.PNG)

**Comment by kartik-budhiraja** [comment]
@jhreis Can I look into this issue if it's available?

**Comment by kylehickinson** [comment]
@kartik-budhiraja Sorry for not getting back to you, you can check it out if this interests you 🙂

**Comment by ghost** [comment]
I'm going to disagree and say that it should be up to the user to decide, and it should be available in settings. I hate how Firefox for iOS doesn't populate logins. And there's no security or privacy enhancement by flashing a banner that says, "Credentials found for this site."

On a phone, every tap and scroll is a hindrance. It's quite slow to accomplish things. I'd much rather have my logins auto-filled, so I can login faster. Instead of fiddling with form inputs, we can just tap "login" and be on our merry way.

On a desktop, the keyboard makes it easy to navigate page and type out the first characters of your username, so the browser can auto-fill. It's not that easy on a phone, and we don't need to have desktop parity in all cases.

An option in settings is a fair compromise rather than having it one way or the other.

Thanks!

Related issues:

https://github.com/brave/brave-ios/issues/1539

https://github.com/brave/brave-ios/issues/571

**Comment by AasthaGupta** [comment]
@jhreis @kylehickinson I'd like to pick this up if it is available?
